### ISSUE TITLE
[BUG] v 2021.5.3 No Water Boost Service anymore

### ISSUE THREAD

**Issue by CaptInsano** [Pyhass/Hive-Custom-Component#97]
**Describe the bug**
The previously available service "hive.boost_hot_water" is no longer available

HomeAssistant 2021.5.1 
Hive Custom Component 2021.5.3

Developer Tools:

![image](https://user-images.githubusercontent.com/332478/117635945-3d71eb80-b178-11eb-9d8a-dfb20b0113d0.png)

![image](https://user-images.githubusercontent.com/332478/117636034-4fec2500-b178-11eb-903d-47bd3c967c97.png)

![image](https://user-images.githubusercontent.com/332478/117636135-6db98a00-b178-11eb-90ea-17bc6e74057a.png)

Please ignore the water boost scripts; they are what I used to control hot water boost. They are broken now that there are no water services.

Log:

`2021-05-10 10:17:26 ERROR (MainThread) [homeassistant.components.script.hot_water_boost_30_mins] Hot Water Boost 30 mins: Error executing script. Service not found for call_service at pos 1: Unable to find service hive.boost_hot_water
2021-05-10 10:17:26 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/script/__init__.py", line 404, in _async_run
    return await self.script.async_run(variables, context)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1209, in async_run
    await asyncio.shield(run.async_run())
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 345, in async_run
    await self._async_step(log_exceptions=False)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 363, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 563, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1428, in async_call
    raise ServiceNotFound(domain, service) from None
homeassistant.exceptions.ServiceNotFound: Unable to find service hive.boost_hot_water`

**Comment by B-Hartley** [comment]
Agreed. Hot Water Boost service has disappeared in 2021.5.3
I've tried removing and re-adding the integration.
Still no service.

**Comment by ovid-io** [comment]
Yes, seems like after fixing #95 the service is gone altogether

**Comment by ovid-io** [comment]
I have just noticed the following error on HA startup that might be of help:

```
Logger: homeassistant.components.water_heater
Source: custom_components/hive/water_heater.py:56
Integration: Water Heater (documentation, issues)
First occurred: 22:16:06 (1 occurrences)
Last logged: 22:16:06

Error while setting up hive platform for water_heater
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 231, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/hive/water_heater.py", line 56, in async_setup_entry
    platform = entity_platform.async_get_current_platform()
AttributeError: module 'homeassistant.helpers.entity_platform' has no attribute 'async_get_current_platform'
```

**Comment by CaptInsano** [comment]
Fixed in 2021.5.4
Thanks @ovid-io & @KJonline
